### DISCUSSION TITLE
A way to extrude_rotate around a given axis ?

### DISCUSSION THREAD

**Discussion by lcice** [pyvista/pyvista#2785] [Q&A] [answered]
Hi, I would like to know if it was possible to use the extrude_rotate filter around an axis different from the z-axis ? I did not succeed to find any related discussion.

**Comment by tkoyama010** [comment, marked as answer]
We are using a `vtkRotationalExtrusionFilter` for the `extrude_rotate` method. `vtkRotationalExtrusionFilter` can use `SetRotationAxis`, but it is currently not supported on the PyVista side. I have created #2786 to add the feature.
https://github.com/pyvista/pyvista/blob/91a562faaf09342ef4fa705b31126f423c1d6810/pyvista/core/filters/poly_data.py#L2750-L2758

**Reply by lcice** [reply]
Very well, I see. Thank you for your quick answer !
